packages/mobile/src/store/offline-downloads/sagas/watchSaveTrackSaga.ts

```diff
@@ -15,28 +15,29 @@ function* downloadSavedTrack(
   action: ReturnType<typeof tracksSocialActions.saveTrack>
 ) {
   const { trackId } = action
-  const isAllFavoritesDownloadOn = yield* select(
-    getIsCollectionMarkedForDownload(DOWNLOAD_REASON_FAVORITES)
+  const isFavoritesDownloadEnabled = yield* select(
+    getIsFavoritesDownloadsEnabled
   )
 
-  if (!isAllFavoritesDownloadOn) return
-  yield* put(
-    addOfflineItems({
-      items: [
-        {
-          type: 'track',
-          id: trackId,
-          metadata: {
-            favorite_created_at: moment().format('YYYY-MM-DD HH:mm:ss'),
-            reasons_for_download: [
-              {
-                is_from_favorites: true,
-                collection_id: DOWNLOAD_REASON_FAVORITES
-              }
-            ]
+  if (isFavoritesDownloadEnabled) {
```

Comment:
negate to reduce nesting?

Reply:
im down, but in this case its really just an else if at the end so i almost saw it as more clear to "if this case run this small block"?

Reply:
yeah i usually will negate if the next block is long and it continues, but since this is the end, i changed it, not sure why but lol my thoughts